sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -941,7 +943,8 @@ def _write_files_with_auto_sharding(
     destination_files_kv_pc = (
         destination_data_kv_pc
         | 'ToHashableTableRef' >> beam.Map(
-            lambda kv: (bigquery_tools.get_hashable_destination(kv[0]), kv[1]))
+            lambda kv: (bigquery_tools.get_hashable_destination(kv[0]), kv[1])).
```

Comment:
Same as above, but with the additional step of making this lambda a function (you can't add type annotations to lambdas).